frontend/src/context/socket.tsx

```diff
@@ -59,7 +58,9 @@ function SocketProvider({ children }: SocketProviderProps) {
 
     ws.addEventListener("open", (event) => {
       setIsConnected(true);
-      options?.onOpen?.(event);
+      const isNewSession = sessionToken === "NO_JWT";
```

Comment:
Maybe would be better to replace `"NO_JWT"` with `null` as it explicitly represents the absence of a value, WDYT?

Reply:
We seem to have informal constants `NO_JWT` and `NO_GITHUB` defined here: https://github.com/All-Hands-AI/OpenHands/blob/27716171bf56edf2f539b5da4becf0cec066937b/openhands/server/listen.py#L316

Reply:
I'll look into replacing these with `nulls`

Reply:
Unfortunately the WebSocket library does not seem to support nulls here - my guess is that is why these constants were chosen.

Reply:
Yeah they need to be strings sadly